app/routes.py

```diff
@@ -1,2 +1,205 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request
+from flask import request, Blueprint, make_response
+from flask import jsonify
```

Comment:
we can put these all together:

```suggestion
from flask import request, Blueprint, make_response, jsonify
```